examples/bridge-app/telink/src/AppTask.cpp

```diff
@@ -357,6 +358,14 @@ EmberAfStatus HandleReadBridgedDeviceBasicAttribute(Device * dev, chip::Attribut
         MutableByteSpan zclNameSpan(buffer, maxReadLength);
         MakeZclCharString(zclNameSpan, dev->GetName());
     }
+    else if ((attributeId == FeatureMap::Id) && (maxReadLength == 4))
+    {
+        *buffer = (uint32_t) ZCL_BRIDGED_DEVICE_BASIC_INFORMATION_FEATURE_MAP;
```

Comment:
This is wrong, exactly in the way it was wrong in the initial PR; see https://github.com/project-chip/connectedhomeip/pull/25798#discussion_r1146679558

Please look at the actual types of the things involved here and think about what this does.

What really confuses me is that your code for the temperature sensor is right, while this should-be-the-same code is wrong....

Reply:
Ok, I hope this time I got it right. Modified it in latest commit